R/utilities.R

```diff
@@ -129,46 +129,27 @@ get_coding_ssm_status = function(gene_symbols,
 
   if(missing(gene_symbols)){
     message("defaulting to all lymphoma genes")
-    gene_symbols = pull(lymphoma_genes,Gene)
```

Comment:
Definitely makes sense to require the metadata as an argument here to avoid calling this many times.